unsupported/Eigen/CXX11/src/Tensor/TensorContraction.h

```diff
@@ -408,7 +408,7 @@ struct TensorContractionEvaluatorBase
     }
   }
 
-  EIGEN_DEVICE_FUNC void evalTo(Scalar* buffer) const {
+    EIGEN_DEVICE_FUNC void evalTo(Scalar* buffer) const {
```

Comment:
please remove whitespace

Reply:
done

Reply:
@deven-amd it seems you haven't pushed latest commits to address review comments?